app/questionnaire_state/state_answer.py

```diff
@@ -23,21 +23,19 @@ def update_state(self, user_input):
         self.value = self.schema_item.get_user_input(user_input)
         self.other = self.schema_item.get_other_value(user_input)
         if self.schema_item.type == 'Radio' and self.input:
-            self._restore_other_value(user_input)
+            self.restore_other_value(user_input)
 
-    def _restore_other_value(self, user_input):
+    def restore_other_value(self, user_input):
```

Comment:
It was changed so that I could unit test its behaviour, but you're right, it doesn't need to be public. I'll revert this change.